.gitattributes

```diff
@@ -1 +1,3 @@
 generateapis.sh eol=lf
+*.csproj text eol=lf
```

Comment:
I would rather not make this change in this PR. We may well want to do it, but it's not part of the tool.